icebergShaded/generate_iceberg_jars.py

```diff
@@ -124,7 +124,7 @@ def generate_iceberg_jars():
         # Search for all glob results
         results = glob.glob(compiled_jar_abs_pattern)
         # Compiled jars will include tests, sources, javadocs; exclude them
-        results = list(filter(lambda result: all(x not in result for x in ["test", "source", "javadoc"]), results))
+        results = list(filter(lambda result: all(x not in result for x in ["test", "sources", "javadoc"]), results))
```

Comment:
```suggestion
        results = list(filter(lambda result: all(x not in result for x in ["test", "source", "javadoc"]), results))
```

Reply:
what's going on here? why this change and then unchange?

Reply:
My delta clone is in `~/source/delta`. This script matches any list entry as true because of `source` in path and excludes everything. Annoying...

Reply:
I had to make this change to compile. Let me know if this makes sense to change permanently to `sources`. it may cause problems for others.

Or make a solution to just look at the listings with relative path.